### PR TITLE
Add a new batching example daml

### PR DESCRIPTION
**Pull request by JoaoSa-da**

### CODE REVIEW

File: sdk/docs/source/daml/code-snippets/Batching.daml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      owner : Party
+      obs: Party
+   where
+      signatory owner
+      observer obs
+
+      choice Foo : ()
+         controller owner
+         do
+         return ()
```
Comment (andreaslochbihler-da) [inline sdk/docs/source/daml/code-snippets/Batching.daml:31]: I'm surprised that indentation complies. Normally, the `do` is indented less than the body. Is this file anywhere checked for compilation by CI?

File: sdk/docs/source/daml/code-snippets/Batching.daml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+template BatchFoo
+   with
+      owner : Party
+   where
+      signatory owner
+
+      choice Run : ()
+         with
```
Comment (andreaslochbihler-da) [inline sdk/docs/source/daml/code-snippets/Batching.daml:40]: Same here. Normally, `with` is indented less than the block.

**Review by andreaslochbihler-da — APPROVED** [review]
The indentation looks odd. Can you double check that this file is tested somewhere in CI? Independently, I'd recommend to follow the usual indentation style of the other Daml files in the same folder.

File: sdk/docs/source/daml/code-snippets/Batching.daml (hunk 1 above)
Comment (JoaoSa-da) [reply sdk/docs/source/daml/code-snippets/Batching.daml:31]: I'd claim that it does, but not exactly sure where. However, I tried it locally with VSC and got no syntax error and compilation worked :).

**Pull request merged**